src/app/events/[slug]/page.tsx

```diff
@@ -59,3 +65,20 @@ export default function EventEntry({ params }: EventProps) {
     </>
   )
 }
+
+function buildCtaArray(
+  externalLink: Event['externalLink'],
```

Comment:
`externalLink` should be also optional right?

Maybe we could extract the type?

```
type EventCtaProps = {
  externalLink?: Event['externalLink']
  lumaCalendarLink?: Event['lumaCalendarLink']
}
```

Reply:
Yes!